server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/UserGroupUtils.java

```diff
@@ -30,4 +30,13 @@ public static List<String> getUserGroups(String userGroups) {
             throw new IncoreHTTPException(Response.Status.BAD_REQUEST, "Could not map provided user-group.");
         }
     }
+
+    public static boolean isAdmin(List<String> groups) {
```

Comment:
I think we already have a function for checking admin in `Authorizer.java`
```
 public boolean isUserAdmin(List<String> userGroups) {
        return (userGroups.contains("incore_admin") || userGroups.contains("incore_ncsa"));
    }
```

Reply:
change all isAdmin boolean to use isUserAdmin in Authorizer.java